politeiad/backendv2/tstorebe/tstore/tstore.go

```diff
@@ -40,9 +40,15 @@ const (
 )
 
 var (
-	_ plugins.TstoreClient = (*Tstore)(nil)
+	_ plugins.TstoreClient = (*TstoreClient)(nil)
 )
 
+// TstoreClient satisfies the plugin TstoreClient interface.
+type TstoreClient struct {
```

Comment:
Couple things here.

- This doesn't need to be exported since the plugins reference the interface, not the struct itself.
- I would define the `tstoreClient` and all of the `tstoreClient` methods in the same file, not in `tstore.go`.  The existing file name is `blobclient.go`.  I typically like to name files for what they contain, so in this case I would name it `tstoreclient.go`, but I'll leave that decision up to you.